docs/painless/painless-guide/painless-datetime.asciidoc

```diff
@@ -544,7 +579,8 @@ indexed documents
 +
 [source,Painless]
 ----
-if (doc.containsKey('start_datetime') && doc.containsKey('end_datetime')) { <1>
+if (doc.containsKey('start_datetime') &&
+        doc.containsKey('end_datetime')) { <1>
```

Comment:
what's the reason for the newline? can it at least be aligned with the opening paren if a newline is necessary?

Reply:
The <1> was getting cut off, and the code snippets are frustrating to read when scrolling horizontally.  I will re-align this.

Reply:
I actually shortened the names a bit to make this all fit on one line as it's far easier to read that way.